### PR TITLE
World border

### PR DESCRIPTION
**Pull request by eirikh1996**
### Describe in detail what your pull request acomplishes
In this pull request, when a craft is moved, it will now check if the new locations of a craft's hitbox is within the world border native to the Minecraft world

### Checklist
- [ ] Proper internationalization
- [ ] Compiled/tested

### CODE REVIEW

File: modules/Movecraft/src/main/java/net/countercraft/movecraft/async/rotation/RotationTask.java [hunk 1]
```diff
@@ -458,6 +461,22 @@ private boolean checkChests(Material mBlock, MovecraftLocation newLoc) {
         return !testMaterial.equals(mBlock) || oldHitBox.contains(aroundNewLoc);
     }
 
+    private boolean withinWorldBorder(MovecraftLocation location) {
```
Comment (cccm5) [inline modules/Movecraft/src/main/java/net/countercraft/movecraft/async/rotation/RotationTask.java:464]: Why can we not just convert the `MovecraftLocation` to a spigot `Location` and simply make use of `WorldBorder::isInside`?

Comment (eirikh1996) [reply modules/Movecraft/src/main/java/net/countercraft/movecraft/async/rotation/RotationTask.java:464]: IsInside does not exist in 1.10.2 spigot @cccm5

**Comment by eirikh1996** [comment]
@cccm5 Requested changes made

File: modules/api/src/main/java/net/countercraft/movecraft/utils/MathUtils.java [hunk 2]
```diff
@@ -168,4 +170,27 @@ public static int positiveMod(int mod, int divisor) {
...
     }
+
+    /**
+     * Checks if a <link>MovecraftLocation</link> is within the border of the given <link>World</link>
+     * @param world the world to check in
+     * @param location the location in the given <link>World</link>
+     * @return true if location is within the world border, false otherwise
+     */
+    @Contract(pure = true)
+    public static boolean withinWorldBorder(World world, MovecraftLocation location) {
```
Comment (cccm5) [inline modules/api/src/main/java/net/countercraft/movecraft/utils/MathUtils.java:181]: Nullability annotations

**Comment by eirikh1996** [comment]
@cccm5 Requested changes made

**Pull request merged**